package.json

```
@@ -5,7 +5,7 @@
   "icon": "img/ruby-test-explorer.png",
   "author": "Connor Shea <connor.james.shea@gmail.com>",
   "publisher": "connorshea",
-  "version": "0.4.6",
+  "version": "0.5.0",
```

Let's keep it at 0.4.6, I'll bump it when I release a new version
```suggestion
  "version": "0.4.6",
```